.github/workflows/ci.yml

```diff
@@ -167,133 +167,176 @@ jobs:
 
   smoke-test:
     needs: check
+
     strategy:
       matrix:
         os: [ubuntu-latest, windows-latest]
-      fail-fast: true
-    name: 👀 Smoke test / ${{ matrix.os }} / node 18 latest
+
+    name: 🔄 Smoke test / ${{ matrix.os }} / node 18 latest
     runs-on: ${{ matrix.os }}
+
     env:
       REDWOOD_CI: 1
       REDWOOD_VERBOSE_TELEMETRY: 1
       # This makes sure that playwright dependencies are cached in node_modules.
       PLAYWRIGHT_BROWSERS_PATH: 0
+
     steps:
       - uses: actions/checkout@v3
-      - name: 🧶 Set up job
-        uses: ./.github/actions/set-up-job
 
-      - name: 🌲 Setup test project
-        id: setup_test_project
-        uses: ./.github/actions/setup_test_project
+      - name: ⬢ Set up Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: 18
+
+      - name: 🐈 Set up yarn cache
+        uses: ./.github/actions/set-up-yarn-cache
+
+      - name: 🐈 Yarn install
+        run: yarn install --inline-builds
```

Comment:
inline builds will let us debug linking errors. They're not as common as they used to be, but we used to have with `@vscode/ripgrep`